js2-mode.el

```diff
@@ -7264,7 +7264,7 @@ are ignored."
            when (and (eq 'block (js2-comment-node-format node))
                      (save-excursion
                        (goto-char (js2-node-abs-pos node))
-                       (looking-at "/\\*global ")))
+                       (looking-at "/\\*[ ]?global ")))
```

Comment:
There's no need for the square brackets here. Maybe it should be ` *`, though. Does ESLint accept multiple spaces before `global`?